rebar.config

```diff
@@ -3,12 +3,11 @@
 
 {deps,
  [{ranch, "",
-   {git, "https://github.com/ninenines/ranch.git", {tag, "1.1.0"}}}
+   {git, "https://github.com/ninenines/ranch.git", {tag, "1.5.0"}}}
```

Comment:
Is 1.5 compatible with the recent version of ranch_proxy_protocol?

Reply:
I haven't checked. My personal requirements were to bring it back up to 1.5.0 in terms of compat, but I can try 1.6.1 to go for newest.

Reply:
I'll throw cowlib in as well.